packages/fx-core/resource/package.nls.json

```diff
@@ -363,6 +363,9 @@
   "core.createProjectQuestion.OpenAIPluginDomain.placeholder": "Enter your website domain or manifest URL",
   "core.createProjectQuestion.ApiKey": "Enter API Key in OpenAPI Description Document",
   "core.createProjectQuestion.ApiKeyConfirm": "Teams Toolkit will upload the API key to Teams Developer Portal. The API key will be used by Teams client to securely access your API in runtime when API-Based Message Extension is triggered. Teams Toolkit will not store your API key.",
+  "core.createProjectQuestion.OauthClientId": "Enter client id for Oauth registration in OpenAPI Description Document",
+  "core.createProjectQuestion.OauthClientSecret": "Enter client secret for Oauth registration in OpenAPI Description Document",
+  "core.createProjectQuestion.OauthClientSecretConfirm": "Teams Toolkit will upload the client id/secret for Oauth Registration to Teams Developer Portal. The client id/secret will be used by Teams client to securely access your API in runtime when API-Based Message Extension is triggered. Teams Toolkit will not store your client id/secret.",
```

Comment:
Revised:
Teams Toolkit uploads the client id/secret for Oauth Registration to Teams Developer Portal. It is used by Teams client to securely access your API at runtime when API-Based Message Extension is triggered. Teams Toolkit doesn't store your client id/secret.

Reply:
Updated, thanks!